City-of-Pori/pori-kada
# PORIJATE-5 Added feature for waste search, added waste type taxonomy

To test:

1. drush updb -y && drush fra -y && drush cc all
2. Make sure there is Pori waste search feature on
3. Make sure there is Waste type taxonomy
